doc/WebSite/Entity-History.md

```diff
@@ -71,32 +74,90 @@ You can add your selectors in your module's PreInitialize method.
 ### Enable/Disable by attributes
 
 While you can select tracked entities by configuration, you can use the
-**HistoryTracked** and **DisableHistoryTracking** attributes for a single
+**Audited** and **DisableAuditing** attributes for a single
 **entity** or an individual **property**. Example:
 
-    [HistoryTracked]
+    [Audited]
     public class MyEntity : Entity
     {
         public string MyProperty1 { get; set; }
 
-        [DisableHistoryTracking]
+        [DisableAuditing]
         public int MyProperty2 { get; set; }
 
         public long MyProperty3 { get; set; }
     }
 
 All properties of MyEntity are tracked except MyProperty2 since it's
-explicitly disabled. The HistoryTracked attribute can be used to
+explicitly disabled. The Audited attribute can be used to
 save change logs for a desired property.
 
-**DisableHistoryTracking** can be used for an entity or a single **property of an
+**DisableAuditing** can be used for an entity or a single **property of an
 entity**. Thus, you can **hide sensitive data** in change logs, such as
 passwords for example.
 
+### Reason
+
+Entity change set has a **Reason** property that can be used to further group
+change sets. Since each SaveChanges call creates an entity change set, a single
+request can result in more than one change set.
```

Comment:
``Reason`` property actually not for grouping change sets. It can be side effect if we use same reason. But notice that same URL can be requested in different times by different users/tenant. So, they are not actually so related together.

``Reason`` property is actually is used to understand why this change has occured. What was the use case made this change. For example, I transferred money from Account A to Account B. Both account balance changes. And I see the Reason as "money transfer" when I check the change set. But a balance change can be other reasons too. ``Reason`` property makes me know why it's done.

Reply:
Great explanation!